articles/azure-functions/app-service-export-api-to-powerapps-and-flow.md

```diff
@@ -45,11 +45,11 @@ To export the API definition, follow these steps:
 
     ![Export to PowerApps + Microsoft Flow button](media/app-service-export-api-to-powerapps-and-flow/export-apps-flow.png)
 
-3. Select the **Export mode**:
+3. Select the **Export Mode**:
 
-    **Express** lets you create the custom connector from within the Azure portal. It requires that you are signed into PowerApps or Microsoft Flow and have permission to create connectors in the target environment. This is the recommended approach if that requirement can be met. If using this mode, follow the [Use express export](#express) instructions below.
+    **Express** lets you create the custom connector from within the Azure portal. It requires that you are signed into PowerApps or Microsoft Flow and have permission to create connectors in the target environment. This is the recommended approach if those requirements can be met. If using this mode, follow the [Use express export](#express) instructions below.
```

Comment:
Two requirements must be met, so "those requirements" instead of "that requirement"